proto/exocore/oracle/v1/genesis.proto

```diff
@@ -32,10 +35,18 @@ message GenesisState {
   repeated RecentMsg recent_msg_list = 6[(gogoproto.nullable) = false]; 
   // cached recent params
   repeated RecentParams recent_params_list = 7[(gogoproto.nullable) = false];
+
+  // information for NST related
   // stakerInfos for each nst token
   repeated StakerInfosAssets staker_infos_assets = 8[(gogoproto.nullable) = false];
   // stakerList for each nst token
   repeated StakerListAssets staker_list_assets = 9[(gogoproto.nullable) = false];
+
+  // information for slashing history
+  // ValidatorReportInfo records all the validatorReportInfos
+  repeated ValidatorReportInfo report_infos = 10[(gogoproto.nullable)=false];
```

Comment:
keep this two fields the  same name formate as the above
_validator_report_infos
validator_missed_rounds_